README.md

```diff
@@ -1,70 +1,90 @@
+This new implementation loops over all the fields exposed in diagnostic commands and tries to get data from them.
 
+Currently, these metric sources are implemented:  
+- $collStats  
+- $indexStats
+- getDiagnosticData  
+- replSetGetStatus  
+- serverStatus  
 
-### Starting the sandbox
-The testing sandbox starts n MongoDB instances as follow:
-- 3 Instances for shard 1 at ports 17001, 17002, 17003
-- 3 instances for shard 2 at ports 17004, 17005, 17006
-- 3 config servers at ports 17007, 17008, 17009
-- 1 mongos server at port 17000
-- 1 stand alone instance at port 27017
-All instances are currently running without user and password so for example, to connect to the **mongos** you can just use:
-```
-mongo mongodb://127.0.0.1:17001/admin
-```
-The sandbox can be started using the provided Makefile using: `make test-cluster` and it can be stopped using `make test-cluster-clean`
-
-To run the unit tests, just run `make test`
+## Flags  
+|Flag|Description|Example|  
+|-----|-----|-----|  
+|-h, \-\-help|Show context-sensitive help||  
+|\-\-compatible-mode|Exposes new metrics in the new and old format at the same time||
+|\-\-mongodb.collstats-colls|List of comma separated databases.collections to get stats|\-\-mongodb.collstats-colls=testdb.testcol1,testdb.testcol2|  
+|\-\-mongodb.indexstats-colls|List of comma separated database.collections to get index stats|\-\-mongodb.indexstats-colls=db1.col1,db1.col2|
+|\-\-mongodb.dsn|MongoDB connection URI|\-\-mongodb.dsn=mongodb://user:pass@127.0.0.1:27017/admin?ssl=true|  
+|\-\-expose-path|Metrics expose path|\-\-expose-path=/metrics_new|  
+|\-\-expose-port|HTTP expose server port|\-\-expose-port=9216|  
+|-D, --debug|Enable debug mode||  
+|--version|Show version and exit|  
+  
+### Running the exporter  
+It you built the exporter using the method mentioned in the previous section, the generated binaries are in `mnogo_exporter_linux_amd64/mnogo_exporter` or `mnogo_exporter_darwin_amd64/mnogo_exporter`  
+  
+#### Example: running the exporter connecting to sandbox's primary  
+```  
+mnogo_exporter_linux_amd64/mnogo_exporter --mongodbdsn=mongodb://127.0.0.1:17001  
+```  
+#### Enabling collstats metrics gathering  
+`--mongodb.collstats-colls` receives a list of databases and collections to monitor using collstats.  
+Usage example: `--mongodb.collstats-colls=database1.collection1,database2.collection2`
 
-### Build the exporter
-The build process uses the dockerized version of goreleaser so you don't need to install Go.
-Just run `make build` and the new binaries will be generated under the build directory.
+#### Enabling compatibility mode.
+When compatibility mode is enabled by the `--compatible-mode`, the exporter will expose all new metrics with the new naming and labeling schema and at the same time will expose metrics in the version 1 compatible way.   
+For example, if compatibility mode is enabled, the metric `mongodb_ss_wt_log_log_bytes_written` (new format) 
```

Comment:
written became unwritten, Is it expected behaviour or mistake?

Reply:
It is intentional